project/projecj_proposal.md

```diff
@@ -0,0 +1,5 @@
+# PyTorch-NLP CONTRIBUTION
```

Comment:
this file describes an existing project, not your new project.

add description of your library and move to `projects/python/pytorch-nlp-chatbot.md`